mmv1/products/compute/terraform.yaml

```diff
@@ -718,10 +718,26 @@ overrides: !ruby/object:Overrides::ResourceOverrides
           mig_template_name: "l7-ilb-mig-template"
           hc_name: "l7-ilb-hc"
           mig_name: "l7-ilb-mig1"
-          fw-allow_iap_hc_name: "l7-ilb-fw-allow-iap-hc"
+          fw_allow_iap_hc_name: "l7-ilb-fw-allow-iap-hc"
```

Comment:
If we change the variable here, we'll also need to change the variable at https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/templates/terraform/examples/internal_http_lb_with_mig_backend.tf.erb#L151 since that is still using the version with the hyphen

Reply:
done